bin/tx-prover/src/error.rs

```diff
@@ -0,0 +1,23 @@
+use axum::http::uri::InvalidUri;
+use thiserror::Error;
+
+// TX PROVER ERROR
+// ================================================================================================
+
+#[derive(Debug, Error)]
+pub enum TxProverProxyError {
```

Comment:
nit: should we rename this into something like `TxProverServiceError`?

Reply:
At the moment it is used only in proxy-related things. Though it can be good having it as a more generic error for the whole binary. I will change it.